airbyte-cdk/python/unit_tests/sources/streams/http/requests_native_auth/test_requests_native_auth.py

```diff
@@ -163,6 +163,12 @@ def test_refresh_access_token(self, mocker):
 
         assert ("access_token", 1000) == token
 
+        # Test with expires_in as str
+        mocker.patch.object(resp, "json", return_value={"access_token": "access_token", "expires_in": "2000"})
+        token = oauth.refresh_access_token()
+
+        assert ("access_token", 2000) == token
+
```

Comment:
Expanded the unit tests with another assertion. Not sure if this is conventional in Airbyte repo, or new test should be added. LMK.